content/references/rippled-api/public-rippled-methods/public-rippled-methods.md

```diff
@@ -97,6 +97,14 @@ Use these methods to perform convenient tasks, such as ping and random number ge
 * **[`ping`](ping.html)** - Confirm connectivity with the server.
 * **[`random`](random.html)** - Generate a random number.
 
+## [NFT Methods](nft-methods.html)
```

Comment:
This is a reasonable way of categorizing the methods, but I would put it above the "Utility methods" and "Server Info Methods" because these are closer in purpose to the methods above those. Alternatively, you could add the 3 methods to existing categories: `account_nfts` would go with the other "Account Methods" (which makes sense, given that it starts with `account_` and functions similarly to those); and the buy/sell methods could go in the "Path and Order Book Methods" category, since they provide the same function for NFTs as the `book_offers` method does for fungible tokens.